15th Small Improvments

zumbak04 opened the pull request:
<!--
A basic changelog, goes to patch notes of the next version, so should be as short and informative as possible.
-->
## Changelog:
- Allowed bastards in all the mod bloodlines except Cenarius, Tyrande, Malfurion and Silver Hand Order bloodlines.
- **Viking**, **Pirate**, **Ravager**, **Sea King/Queen** give less **Southern Opinion**.
- **Blood Elf** race gives 1 **Martial** and 1 **Learning** now.
- Non-adult females no longer betroth lowborn males after a race-incompatible betrothal. Closes #654.
- If you organize a race-incompatible betrothal, the fiance(e) and his/her relatives get **Broken Betrothal** opinion of you instead of **Divorced** opinion. Closes #654.
- Tribals and nomads can join **the Cult of the Damned**, **Twilight's Hammer Cult** and **Shadow Council** without following **Necromantic**, **Void**, **Fel** religions. Closes #906.
- Vampires can abduct courtiers without being in a society. Closes #946.

<!--
A changelog where you can describe more complicated things to other developers and testers.
-->
## Scripting changelog (ignore if you can't read CK2 script):
- `WCCLN.130` prefers to pick a lowborn-lover to be your spouse.
- Fixed an issue where you could abduct a landless duke or above.

<!--
If you need to explain something to testers. Otherwise, delete this part.
-->
## How to test:
Just a few notes to save your time:
- `add_trait viking` to add Viking trait.
- `add_trait pirate` to add Pirate trait.
- `add_trait ravager` to add Ravager trait.
- `add_trait seaking` to add Sea King trait.
- `add_trait sea_queen` to add Sea Queen trait.
- `add_trait creature_blood_elf` to add Blood Elf trait.
- Chief Lucard of Ominous Island is a vampire near Zandalar.

## Notes:
A race-incompatible betrothal/marriage is a situation when you betroth/marry 2 incompatible races.

MemeWizard commented on decisions/mnm_society_decisions.txt:
```diff
@@ -2717,10 +2709,13 @@ targeted_decisions = {
 			OR = {
 				lower_tier_than = DUKE
-				is_landed = no
+				is_ruler = no
```
Don't forget to put in change log that you fixed vanilla bug, since your character could abduct landless character above duke tier before.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/75931989-5c476480-5e87-11ea-8846-0b0cbd458fac.png)

</details>

zumbak04 replied:
Mentioned in the scripting changelog.

MemeWizard commented on decisions/mnm_society_decisions.txt:
```diff
@@ -2756,64 +2751,106 @@ targeted_decisions = {
-						has_society_currency_tiny_trigger = yes
+			
+			# Warcraft
+			trigger_if = {
```
I didn't get it. Isn't conditions are the same?

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/75932529-a8df6f80-5e88-11ea-8b56-de7d9475ebaa.png)

</details>

zumbak04 replied:
The first if is shown if you have society currency. If you don't have, vampires see their own condition, non-vampires see the else trigger.

MemeWizard replied:
Add comment about it, pls

zumbak04 replied:
Added in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/b8fc8f8ef0aa4893f016a1018991f26406def8bd.

MemeWizard commented on decisions/mnm_society_decisions.txt:
```diff
@@ -2756,64 +2751,106 @@ targeted_decisions = {
 		}
 		
 		effect = {
+			# Warcraft
```
Also didn't get it here

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/75932611-e9d78400-5e88-11ea-9f40-ca733acadbcf.png)

</details>

MemeWizard commented:
Thanks for review <3

> Is this intended? If yes, why?

It's intended to change (#1273), just not in this PR.

> Able to join secret societies as Tribal/Nomadic, however conditional text only appears if you don't meet the requirements to join.

Intended

> Also, the hard limitation of evil religion = appropriate society is still in place.

> Tribals and nomads can join the Cult of the Damned, Twilight's Hammer Cult and Shadow Council without religious conditions.

That changelog is misleading, you think that your character don't need to care about religion if it's nomadic or tribal in order to join evil societies, but actually, this changelog says that before this PR, tribal or nomadic character could join society only if they already have appropriate evil religion... Something like that.

zumbak04 commented:
> What society

The shackles of the society have been broken in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1274/commits/eaf6a7ea4682d83ebfb1ff06fa7956c648cb9960.

> Are you sure that you want to allow all bloodlines to be inherited by bastards?
> In vanilla, for example, all saintly bloodlines do not allow bastard inheritance.

Changed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1274/commits/d5d71b0851bb1cb77cbadc20c88e3ae41cf36632.

> Able to join secret societies as Tribal/Nomadic, however conditional text only appears if you don't meet the requirements to join. Also, the hard limitation of evil religion = appropriate society is still in place.

Changed the changelog, I hope it's more clear now.

MemeWizard commented:
For oblivious reasons, I suggest to replace `is_in_society = yes` with "member of society that can abduct characters" (new scripted trigger?).

MemeWizard commented on localisation/00_zuwc_united_localisation.csv:
```diff
@@ -2881,13 +2881,13 @@ is_dragon_race_trigger_tooltip;Is a dragon;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+text_effect_decision_abduct;Enables �GAttempt to Abduct�! on character portraits;;;;;;;;;;;;;x
```
Maybe it should be clarified in text effect that vampires can abduct only courtiers?

zumbak04 replied:
Mentioned in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1274/commits/a002eeccb6ee23b0c75842394c70c88bf7222781.

zumbak04 commented:
> For obvious reasons, I suggest to replace `is_in_society = yes` with "member of society that can abduct characters" (new scripted trigger?).

Added in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1274/commits/155ff5752be92db2ff0499dd27689e18a96676a0.

MemeWizard commented:
> Added

Ok, looks like reasons wasn't obvious enough.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/76142294-400d1880-607d-11ea-8a8c-81cbe50026f7.png)
![image](https://user-images.githubusercontent.com/46576860/76142299-4a2f1700-607d-11ea-9650-0fa8add34e31.png)
![image](https://user-images.githubusercontent.com/46576860/76142304-5915c980-607d-11ea-96a3-8faa9659ae78.png)


</details>

zumbak04 commented:
> Ok, looks like reasons wasn't obvious enough.

Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1274/commits/78a482b4554f8e0c3c76f387ba6876fb12353bef.
